src/goto-symex/symex_assign.cpp

```diff
@@ -232,7 +232,8 @@ void goto_symext::symex_assign_symbol(
     ssa_rhs,
     ns,
     options.get_bool_option("simplify"),
-    constant_propagation);
+    constant_propagation,
+    options.get_bool_option("java-threading"));
```

Comment:
To avoid repeated string lookups, could you please add a const member to `goto_symext` that is set in the constructor? (The same should be done for "simplify" above, but that's not something you've newly introduced.)

Reply:
done, see  https://github.com/diffblue/cbmc/pull/2354